LCM/scripts/InstallModule.py

```diff
@@ -256,6 +256,14 @@ def main(args):
 
         pythonVersionFileNames = ['2.4x-2.5x', '2.6x-2.7x', '3.x']
 
+        # Python 2.4 and 3 recognize different formats for octal
```

Comment:
This needs to move before the resource loop so that it is actually before the loop and doesn't run for each resource.
Before this line: "for resource in moduleResources:"